airbyte-webapp/src/views/Connector/ConnectorForm/index.stories.tsx

```diff
@@ -216,5 +215,142 @@ Oneof.args = {
+      title: "Default Dataset ID",
+    },
+    dataset_location: {
+      type: "string",
+      description: "The location of the dataset. Warning: Changes made after creation will not be applied.",
+      title: "Dataset Location",
+      default: "US",
+      enum: [
+        "US",
+        "EU",
+        "asia-east1",
+        "asia-east2",
+        "asia-northeast1",
+        "asia-northeast2",
+        "asia-northeast3",
+        "asia-south1",
+        "asia-southeast1",
+        "asia-southeast2",
+        "australia-southeast1",
+        "europe-central1",
+        "europe-central2",
+        "europe-north1",
+        "europe-west1",
+        "europe-west2",
+        "europe-west3",
+        "europe-west4",
+        "europe-west5",
+        "europe-west6",
+        "northamerica-northeast1",
+        "southamerica-east1",
+        "us-central1",
+        "us-east1",
+        "us-east4",
+        "us-west-1",
+        "us-west-2",
+        "us-west-3",
+        "us-west-4",
+      ],
+    },
+    credentials_json: {
+      type: "string",
+      description:
+        'The contents of the JSON service account key. Check out the <a href="https://docs.airbyte.io/integrations/destinations/bigquery">docs</a> if you need help generating this key. Default credentials will be used if this field is left empty.',
+      title: "Credentials JSON",
+      airbyte_secret: true,
+    },
+  },
+};
+
+export const Preview: StoryObj<React.FC<React.ComponentProps<typeof ConnectorForm> & PreviewProps>> = {
+  render: (props) => {
+    const connectorDefinitionSpecification: ConnectorDefinitionSpecification = {
+      ...TempConnector,
+      destinationDefinitionId: "some-id",
+      jobInfo: {
+        id: "some-id",
+        createdAt: Date.now(),
+        endedAt: Date.now() + 1,
+        configType: "get_spec",
+        succeeded: true,
+      },
+      connectionSpecification: props.connectionSpecification,
+    };
+    return (
+      <ConnectorForm
+        renderWithCard
+        formType={props.formType}
+        renderFooter={() => (
+          <div>
+            <Button type="submit">Submit form</Button>
+          </div>
+        )}
+        onSubmit={props.onSubmit}
+        selectedConnectorDefinitionSpecification={connectorDefinitionSpecification}
+      />
+    );
+  },
+  args: {
+    connectionSpecification: json,
+  },
+  parameters: {
+    actions: {
+      onSubmit: { action: "submit" },
+    },
+  },
+  argTypes: {
+    formValues: hidden,
+    renderWithCard: hidden,
+    title: hidden,
+    description: hidden,
+    full: hidden,
+    formId: hidden,
+    selectedConnectorDefinition: hidden,
+    selectedConnectorDefinitionSpecification: hidden,
+    isEditMode: hidden,
+    connectorId: hidden,
+    connectionTestSuccess: hidden,
+    isTestConnectionInProgress: hidden,
+    bodyClassName: hidden,
+    headerBlock: hidden,
+    renderFooter: hidden,
```

Comment:
:information_source: By default all properties of the base component (default export) will show up in storybook, so hiding all except `onSubmit` (needed for Actions to work), and the `formType` one (just to be able to render it nicely in whatever type you prefer). Also adding the `connectionSpecification` property here, which our custom render function on top will use.